x-pack/plugins/task_manager/server/saved_objects/migrations.ts

```diff
@@ -142,6 +145,29 @@ function moveIntervalIntoSchedule({
   };
 }
 
+function resetUnrecognizedStatus(
```

Comment:
@ersin-erdal We will have to keep an eye out when merging this PR and [yours](https://github.com/elastic/kibana/pull/123963). One of us will have to use `pipeMigrations` to run both migrations for 8.2 :)